Sources/WebSocketKit/PMCE.swift

```diff
@@ -444,15 +340,13 @@ public final class PMCE: Sendable {
             self.decompressorBox = NIOLoopBoundBox( CompressionAlgorithm.deflate(configuration: zcdConf).decompressor,
                                              eventLoop: channel.eventLoop)
             
-            logger.trace("compressor \(zccConf)")
-            logger.trace("decompressor \(zcdConf)")
+           
         }
         startStreams()
     }
     
-    ///  Starts compress-nio streams for DEFLATE support.
-    /// - returns: Void
-    public func startStreams() {
+   
+    func startStreams() {
```

Comment:
Why not private?

Reply:
I made start and stop streams private in my last commit.